src/pages/donate.astro

```diff
@@ -0,0 +1,165 @@
+---
+
+import Layout from '../layouts/Layout.astro';
+import Container from "../components/layout/Container.astro";
+import ExternalLogo from "../components/layout/ExternalLogo.astro";
+
+---
+
+<Layout title="Donate">
+  <div class="bg-nixdarkblue px-12 pt-16 pb-16 md:pb-0 text-white">
+    <Container class="grid px-12">
+      <h1 class="text-5xl font-heading font-bold leading-none text-center md:text-left">
+        Donate
+      </h1>
+    </Container>
+  </div>
+  <img src="/divider/mountain_nixdarkblue.svg" class="w-full hidden md:block" />
+
+  <Container class="grid px-12 pt-16">
+    <p class="text-xl font-bold text-gray-500">
+      The NixOS infrastructure, development and events are funded by its users and sponsors.
+    </p>
+  </Container>
+
+  <div class="flex flex-col">
+
+    <Container class="grid md:grid-cols-2 gap-8 px-12 py-16">
+      <div class="flex flex-col justify-center gap-4">
+        <p class="font-light">
+          Maintaining our build farm and binary cache is very expensive.
+          We depend on your support to ensure the continuous operation and expansion of the NixOS infrastructure.
+        </p>
+        <p class="font-light">
+          Please consider donating a small amount every month to contribute to sustainable funding.
+          Donations are handled by a nonprofit organization, the <a href="/governance.html#foundation">NixOS Foundation</a>.
+        </p>
+      </div>
+      <div class="flex flex-col gap-4">
+        <img src="/images/logos/nixos_flake.svg" class="h-40" />
+        <p class="text-center font-nixos font-regular text-8xl">
```

Comment:
since the font is just used in this logo, I think it would be a better choice to just do an SVG with an alt text containing the text as curves so save up on import logic and assets

Reply:
I did find one other spot that uses the Vegur font. Regardless, I would like to keep the font for now. It is easier to adjust font sizes so things align and look correct. Maybe another PR in the future?